sdk/purview/Azure.Analytics.Purview.Scanning/docs/PurviewClassificationRuleClient.xml

```diff
@@ -0,0 +1,689 @@
+<Type Name="PurviewClassificationRuleClient" FullName="Azure.Analytics.Purview.Scanning.PurviewClassificationRuleClient">
+  <TypeSignature Language="C#" Value="public class PurviewClassificationRuleClient" />
+  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit PurviewClassificationRuleClient extends System.Object" />
+  <TypeSignature Language="DocId" Value="T:Azure.Analytics.Purview.Scanning.PurviewClassificationRuleClient" />
+  <TypeSignature Language="VB.NET" Value="Public Class PurviewClassificationRuleClient" />
+  <TypeSignature Language="F#" Value="type PurviewClassificationRuleClient = class" />
+  <AssemblyInfo>
+    <AssemblyName>Azure.Analytics.Purview.Scanning</AssemblyName>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
```

Comment:
Since we are not using these XML files under `Generated/docs` folder for E2E testing, can we delete these XML files?

Reply:
sure